scripts/jenkins/red_hat/red_hat_docker_certification.py

```diff
@@ -22,10 +22,18 @@ def has_scan_errors(digest, pid, api_key):
     response = None
     while response is None:
         intermediate_response = requests.get(url=url, headers=headers)
+        print("Intermediate response: {}".format(intermediate_response.status_code))
         if intermediate_response.status_code == 200:
-            response = intermediate_response
+            try:
+                intermediate_response.json()["data"]["results"]
+                response = intermediate_response
+            except (KeyError, TypeError):
+                # Do effectively nothing if image check results are missing in the response
+                # There are various states the Red Hat API responses with 200, yet the payload may differ
+                # causing KeyError or TypeError when trying to access scan results
+                response = None
         else:
-            time.sleep(5)
+            time.sleep(30)
```

Comment:
any reason why?

Reply:
It takes 30 minutes approximately - up to 4 hours officially. No need to poll it every 5 seconds or so. Or is it ?

Reply:
that is a good reason 👍